installer/roles/kubernetes/tasks/kubernetes.yml

```diff
@@ -0,0 +1,16 @@
+- name: Set the Kubernetes Context
+  shell: "kubectl config set-context {{ kubernetes_context }}"
+
+- name: Get Namespace Detail
+  shell: "kubectl get namespace {{ awx_kubernetes_namespace }}"
+  register: namespace_details
+  ignore_errors: yes
+
+- name: Create AWX Kubernetes Project
+  shell: "kubectl create namespace {{ awx_kubernetes_namespace }}"
```

Comment:
{{ kubectl_bin }} ?

Reply:
Doesn't get defined until after this file is executed. Because I need to pivot between using `oc` or `kubectl` in further down in the role. Maybe `kubectl_or_oc` would be a better name?

Reply:
Done